tools/test_data_gen.py

```
@@ -0,0 +1,48 @@
+    dump_all_diags(
+        test_restarts.setup_model().populations, timestamps=restartable_sim.timestamps
+    )
+    Simulator(sim).run()
+
+
+def launch(fn, n=5):
+    """Launch secondary process to run first simulation to avoid initalizing MPI now"""
+    cmd = f"mpirun -n {n} python3 -O {__file__} {fn}"
+    print(cmd)
+    try:
+        p = subprocess.run(cmd.split(" "), check=True, capture_output=True)
```

Ruff

Local variable `p` is assigned to but never used

Remove assignment to unused variable `p`

(F841)